docs/submission/submitting-molecular-data.md

```diff
@@ -57,78 +57,161 @@ storage.url=https://score.qa.argo.cancercollaboratory.org
+
+The data fields can be broken down into four main sections: `experiment`, `sample`, `read groups` and `files`.
+
+> Note: \*\* denotes a required field. These must be provided as part of the metadata payload or it will immediately fail validation upon submission.
+
+#### **Experiment:** The experiment section contains details relevant the experimental requirements imposed during sequencing.
+
+- `**studyId`: Corresponds to your ARGO `Program ID`. This is the same Program ID that you see when you log in to the ARGO Data Platform.
+- `**submitter_sequencing_experiment_id`:The unique identifier of the sequencing experiment.
+- `**platform`: The sequencing platform type that was used to generate the submitted data files.
+- `platform_model`: The exact model number of the sequencing machine used.
+- `sequencing_center`: The sequencing center the analysis was performed at.
+- `**experimental_strategy`: Descriptor of the read domain experiment method; Sometimes also referred to as library strategy.
+- `sequencing_date`: Date sequencing was performed.
+- `read_group_count`: Number of read groups submitted as part of the raw molecular file.
+
+**Sample:** The sample section contains details of the clinical data and key sample descriptors related to the submitted files. In order to submit a payload, this data must already be [registered](../submission/registering-samples) to the ARGO Data Platform. For allowed values of all fields, please see the Sample Registration file of the [Data Dictionary](/dictionary)
+
+If the information for a sample is different than what has been registered, validation will fail immediately upon submission.
+
+**Read Groups:** The read group section contains details about the reads that were generated from a single run of a sequencing instrument lane. The number of `read_group` objects in the payload must meet the number specified in `read_group_count`.
+
+- `**submitter_read_group_id`: The unique identifier of a read group.
+- `**platform_unit`:
+- `**is_paired_end`: `true` for paired end sequencing, otherwise `false`.
+- `**file_r1`: Name of the read group file to be submitted.
+- `file_r2`: Name of the read group file to be submitted. Required if and only if paired end sequencing was done.
+- `read_length_r1`: Average length of reads in `file_r1`.
+- `read_length_r2`: Average length of reads in `file_r2`.
+- `insert_size`:
+- `sample_barcode`:
+- `**library_name`:
+
+Read Group Data Validations:
+
+1. `submitter_read_group_id` must be unique in within each Song payload, and ideally unique across all read groups in an ARGO program.
+1. `submitter_read_group_id` must not contain any special characters, with the exception of `-` , `.` , and `\_` .
+1. All `read_groups` in the payload must belong to a single sample.
+1. `platform_units` must be unique in metadata with a one-to-one relationship with `submitter_read_group_id`.

```

Comment:
add "the" ?
must be unique in `the` metadata

Reply:
removeing metadata